src/loki/worker.cc

```diff
@@ -60,27 +60,28 @@ void loki_worker_t::parse_locations(google::protobuf::RepeatedPtrField<valhalla:
   }
 }
 
-void loki_worker_t::parse_costing(Api& api) {
+void loki_worker_t::parse_costing(Api& api, bool allow_none) {
```

Comment:
this is how a particular API can tell the costing parser that its ok to have the `none` costing type. only locate allows it so far